charts/timed/values.yaml

```
@@ -72,6 +72,50 @@ backend:
     serverEmail: webmaster@chart-example.local
     # backend.settings.workReportPath -- Path where the workreport shall be loaded from. The contents of the default path is filled from `configmap-workreport.yaml`.
     workReportPath: /etc/workreport
+  cronjobs:
+    # backend.cronjobs.notifySupervisors -- Notify supervisors
+    notifySupervisors:
+      schedule: "0 8 * * 4"
+      command:
+        - "./manage.py"
+        - "notify_supervisors_shorttime"
+    # backend.cronjobs.notifyReviewersFirst -- Notify reviewers first stage
+    notifyReviewersFirst:
+      schedule: "0 8 4 * *"
+      command:
+        - "./manage.py"
+        - "notify_reviewers_unverified"
+        - "--offset 5"
```

You might consider splitting these
```suggestion
        - "--offset"
        # backend.cronjobs.notifyReviewersFirst[3] - some docs about offset
        - "5"
```